README.md

```diff
@@ -10,8 +10,11 @@ The Student Robotics public website.
 
 2. Install Bundler and Rake
 
-    ``` shell
-    $ gem install bundler rake
+    ```
+    $ cat ./scripts/install-gems
```

Comment:
Why are we telling people to `cat` the file before running it?

Reply:
So they know what its going to do. I'm happy to do that if you think it's not useful.

Reply:
I'm pretty sure that most people won't _actually_ cat it.

Reply:
I think i'd rather we still not include this. If people want to know what's going on, they can just open the file themselves.

Reply:
39cdd25